README.md

```diff
@@ -181,6 +181,10 @@ This is the folder structure:
 /.storybook         # storybook config
 ```
 
+### Decryption
+
+In some instances, decryption of a resource must be performed upon receiving it. For this purpose you can pass an optional `getContent` function that will be used to fetch (and possibly decrypt) the resources to be displayed. It is suggested to perform heavy tasks such as decryption in a Web Worker. The example app shows an example of this being done using the private NYPL AxisNow decryptor (not available publicly). It sets up a WebWorker and communication channel using [Comlink](https://github.com/GoogleChromeLabs/comlink) to fetch, decrypt, and re-embed encrypted subresources.
```

Comment:
can you elaborate on what you mean by 
> decryption of a resource must be performed upon receiving it.

Reply:
Does this mean that pretty much all the decryption happens within webreader, under the hood?   Nothing additional for the consuming app to set up?

Reply:
Ok I updated this, lmk what you think